src/tokenlist.schema.json

```diff
@@ -231,19 +231,33 @@
         "name": {
           "type": "string",
           "description": "The name of the token",
-          "minLength": 1,
+          "minLength": 0,
           "maxLength": 40,
```

Comment:
@zzmp btw, this limit isn't correct. You can have more than 40 chars and as far as I know there isn't a limit.

This pair is one https://dexscreener.com/polygon/0x841120E51aD43EfE489244728532854A352073aD

Reply:
Ok